tezos_interop/dune

```diff
@@ -2,7 +2,7 @@
  (name tezos_interop)
  (libraries helpers lwt.unix mirage-crypto-ec tezos-micheline)
  (preprocess
-  (pps ppx_deriving_yojson ppx_blob))
+  (pps ppx_deriving_yojson ppx_deriving.eq ppx_blob))
```

Comment:
eq was added but not used, why is that the case?

Reply:
it was used in Address and Ticket.